### PR TITLE
Wraith's Timeshift Cancels on Z-Level Change

### PR DESCRIPTION
**Pull request by Runian**
## About The Pull Request
If you change z-levels while affected by Timeshift, it is canceled.
Updates Timeshift's description to mention that it is canceled if target changes z-levels.

## Why It's Good For The Game
Using Timeshift on marines who just barely left into orbit on the TAD and suddenly become marooned when they're pulled from orbit to ground side is apparently "bug abuse". So, bug fixing.

## Changelog
:cl:
fix: Wraith's Timeshift cancels if the target changes z-levels.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm [hunk 1]
```diff
@@ -790,3 +783,23 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
...
+		targeted.status_flags &= ~(INCORPOREAL|GODMODE)
+		UnregisterSignal(targeted, COMSIG_MOVABLE_MOVED)
+		UnregisterSignal(targeted, COMSIG_MOVABLE_Z_CHANGED)
+		REMOVE_TRAIT(targeted, TRAIT_TIME_SHIFTED, XENO_TRAIT)
+		targeted = null
+
+/// Cancels the timeshift if the target changed z-levels.
+/datum/action/ability/activable/xeno/rewind/proc/target_changed_zlevel()
+	SIGNAL_HANDLER
+	cancel_timeshift()
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm:805]: why's this a separate proc?

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Comment by iforgotmeword** [comment]
wait me learning to buckle my self into a chair as soon i see the time shift text was just bug abusing and not a skill issue?

**Comment by Arturlang** [comment]
Alas, one PR at a time, wraith moves into obscurity

**Comment by Runian** [comment]
> wait me learning to buckle my self into a chair as soon i see the time shift text was just bug abusing and not a skill issue?

image

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**